packages/core/frontend/sirius-components-core/src/icon/IconOverlay.tsx

```diff
@@ -10,6 +10,7 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
+import { Tooltip } from '@material-ui/core';
```

Comment:
Invalid Material import (detected by the backend tests).

Reply:
Thanks, I changed the import, rebased and also I added tooltip to the DiagramPanel to have the same style